### ISSUE TITLE
podman images --filter dangling=false lists only dangling images

### ISSUE THREAD

**Issue by refi64** [containers/libpod#2884]
/kind bug

**Description**

When you run `podman images -f "dangling=false"`, only dangling images are displayed (aka the result is identical to `dangling=true`).

**Steps to reproduce the issue:**

1. Run `podman images -f "dangling=false"`.

2. Cry.

3. Use `podman images | grep -v '<none>'`? :rofl:

**Describe the results you received:**

Dangling images.

**Describe the results you expected:**

Non-dangling images.

**Additional information you deem important (e.g. issue happens only occasionally):**

Maybe related to #2246?

This is how it looks with `dangling=false`:

```
REPOSITORY   TAG      IMAGE ID       CREATED        SIZE
<none>       <none>   4e462d77aba2   44 hours ago   332 MB
<none>       <none>   40860aca20cf   8 days ago     666 MB
<none>       <none>   2d23bc730f05   6 weeks ago    661 MB
<none>       <none>   cc370c6074e2   6 weeks ago    589 MB
<none>       <none>   e2dbe0d2f88a   6 weeks ago    397 MB
<none>       <none>   cccfcf6bb935   6 weeks ago    321 MB
<none>       <none>   166c269e4b2c   6 weeks ago    178 MB
<none>       <none>   170e5a800794   6 weeks ago    178 MB
<none>       <none>   64a4c15f42c7   6 weeks ago    178 MB
<none>       <none>   c3fe047bee49   2 months ago   102 MB
<none>       <none>   6a2b7a4014f8   2 months ago   102 MB
<none>       <none>   f65bc6a66d10   2 months ago   78.8 MB
<none>       <none>   6c411652b17d   2 months ago   102 MB
<none>       <none>   0a72e0433e87   2 months ago   102 MB
<none>       <none>   3f53bb00af94   3 months ago   4.67 MB
```

**Output of `podman version`:**

```
Version:            1.1.2
RemoteAPI Version:  1
Go Version:         go1.12
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  git commit: ""
  go version: go1.12
  podman version: 1.1.2
host:
  BuildahVersion: 1.7.1
  Conmon:
    package: podman-1.1.2-2.dev.git0ad9b6b.fc30.x86_64
    path: /usr/libexec/podman/conmon
    version: 'conmon version 1.12.0-dev, commit: 880e9bdff2052c09f2a7512828cb274721cf4399'
  Distribution:
    distribution: fedora
    version: "30"
  MemFree: 2952314880
  MemTotal: 8222556160
  OCIRuntime:
    package: runc-1.0.0-75.dev.git751f18d.fc30.x86_64
    path: /usr/bin/runc
    version: |-
      runc version 1.0.0-rc6+dev
      commit: 34cd853bfa9dc792faaae59bbb4d3020225c98c8
      spec: 1.0.1-dev
  SwapFree: 8359243776
  SwapTotal: 8359243776
  arch: amd64
  cpus: 4
  hostname: fedora.refi64.com
  kernel: 5.0.5-300.fc30.x86_64
  os: linux
  rootless: true
  uptime: 52m 26.59s
insecure registries:
  registries: []
registries:
  registries:
  - docker.io
  - registry.fedoraproject.org
  - quay.io
  - registry.access.redhat.com
  - registry.centos.org
store:
  ConfigFile: /home/ryan/.config/containers/storage.conf
  ContainerStore:
    number: 8
  GraphDriverName: overlay
  GraphOptions:
  - overlay.mount_program=/usr/bin/fuse-overlayfs
  GraphRoot: /var/home/ryan/.local/share/containers/storage
  GraphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  ImageStore:
    number: 49
  RunRoot: /run/user/1000
  VolumePath: /var/home/ryan/.local/share/containers/storage/volumes
```

Running on Fedora Silverblue 30.

**Comment by kunalkushwaha** [comment]
Just verified with PR https://github.com/containers/libpod/pull/2297 , which fixes #2246 do not fix this.
It seems, while parsing filter `dangling`,  the value passed along with each filter i.e. `true` | `false` is ignored.

/assign

**Comment by kunalkushwaha** [comment]
Closing this issue, as it is fixed with #2885 , shall be available with v1.3.0 release.
